thirdparty/google_crashpad_client/third_party/mini_chromium/mini_chromium/base/compiler_specific.h

```diff
@@ -79,7 +79,9 @@
 #elif defined(__clang__)
 #define FALLTHROUGH [[clang::fallthrough]]
 #else
+#ifndef FALLTHROUGH
 #define FALLTHROUGH
 #endif
```

Comment:
I think we should rather rename our own macro to MU_FALLTHROUGH (in log.h)

Reply:
Ah, yes, i didn't even check where the conflict is

Reply:
Done. Actually that marco isn't really needed as `// FALLTHROUGH` comment works just fine cross-platform and is used in several places already, whereas that `FALLTHROUGH()` marco is use in only 2 places (and one file)